cli/src/commands/router/commands/compose.ts

```diff
@@ -28,7 +28,9 @@ type Config = {
 
 export default (opts: BaseCommandOptions) => {
   const command = new Command('compose');
-  command.description('Generates the router config locally. The output can be piped to a file.');
+  command.description(
+    'Generates the router config locally. The output can be piped to a file. To generate config for production use please use router fetch instead.',
```

Comment:
> .. To generate config for production use please use router fetch instead.

"Generates a router config from a local composition file. This makes it easy to test your router without a control-plane connection. For production, please use the "router fetch" command"